clients/ui/frontend/nginx.conf

```diff
@@ -0,0 +1,22 @@
+server {
+        listen       8080 default_server;
+        listen       [::]:8080 default_server;
+        server_name  _;
+        root         /usr/share/nginx/html;
+        gzip         on;
+        access_log   /dev/stdout main;
+
+        location / {
+            try_files $uri $uri/ /index.html;
+        }
+
+        location /p/ {
```

Comment:
After giving it a thought, I think that this should be called `v1` and leave out that from the API, why?:

1. You can then switch services in case you have a v2 implementation and rereoute that to other differente service
2. If needed in isolation, you can create proxy pass rules in those services too or use an ingress with rules in deployment (so the notation v1, v2 shouldn't be in the implementation)
3. Leaves a clean path

Reply:
That's just a suggestion btw 😅

Reply:
I'm not sure I understand the use case here? 

I think the versioning should be done directly on the API not on a reverse proxy that's only used in the frontend container. We have the ability to change the proxy address in the future since it's just an implementation detail - the backend API will become harder to change at some point down the line.

It's possible I've completely misunderstood what you're suggesting :)

Reply:
wait, hold on, I don't understand this then, I'm sorry I was thinking about something else, but this is just a proxy rule, if we use for example `/api/v1` in our endpoint, why don't we add the rule to detect `/api/v1`, why do we need a `/p/` here?

Reply:
you're right, we could just match /api/ and redirect that - that probably is cleaner. I used to prefer a visual clue that something's being proxied.